scrapers/koji_scraper.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-License-Identifier: GPL-3.0+
+from scrapers.base import BaseScraper
+from purview.models.koji import KojiBuild, KojiTask
+from purview.models.user import User
+from datetime import datetime
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class KojiScraper(BaseScraper):
+
+    def run(self, since=None):
+        log.info('Connected to TEIID, fetching initial load for koji')
```

Comment:
Incorrect log statement cause the connection only happens once you call `self.teiid.query` for the first time.